.pre-commit-config.yaml

```diff
@@ -1,41 +1,53 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v5.0.0
     hooks:
-    - id: check-yaml
-      args: [--allow-multiple-documents, --unsafe]
-    - id: end-of-file-fixer
-    - id: trailing-whitespace
-      # FIXME: these autogenerate files contain trailing whitespace. Need to fix generator.
-      exclude: 'platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/(location/LocationIndicatorLayer|style/layers/PropertyFactory)\.java'
-- repo: https://github.com/pre-commit/mirrors-clang-format
-  rev: v19.1.4
-  hooks:
-  - id: clang-format
-    files: '.*\.(hpp|cpp|h)'
-    exclude: 'vendor/.*'
-- repo: https://github.com/keith/pre-commit-buildifier
-  rev: 7.3.1
-  hooks:
-    - id: buildifier
-- repo: https://github.com/Mateusz-Grzelinski/actionlint-py
-  rev: v1.7.4.20
-  hooks:
-    - id: actionlint
-      additional_dependencies: [shellcheck-py]
-- repo: https://github.com/nicklockwood/SwiftFormat
-  rev: "0.55.3"
-  hooks:
-    - id: swiftformat
-      args: [--swiftversion, "5.8"]
--   repo: local
+      - id: check-added-large-files
+      - id: check-toml
+      - id: check-yaml
+        args: [ --allow-multiple-documents, --unsafe ]
```

Comment:
We don't use spaces around YAML array elements in any of the other YAML files.

Reply:
ok, fixed